Cargo.toml

```
@@ -19,7 +19,7 @@ default = []
 hyperv = ["tss-esapi"]
 
 [dependencies]
-structopt = "0.3"
+clap = { version = "=4.4.18", features = [ "derive" ] }
```

Just curious, why do we want to maintain at `4.4.18`, rather than just `>=4.4.18, <5.0.0`

as in, why is this not:

```suggestion
clap = { version = "4.4.18", features = [ "derive" ] }
```